### PR TITLE
Add Mathieu Scheltienne to MNE-Python Steering Council

### PR DESCRIPTION
**Pull request by sappelhoff**
https://mne.tools/dev/overview/people.html#steering-council

We may need to update the list of institutional partners, right?

Which institution are you currently working for @mscheltienne? And what would be their logo to be displayed (do you have a file for us, or a link to a file?)?

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

**Comment by mscheltienne** [comment]
Thank you for taking the time! 

> We may need to update the list of institutional partners, right?

It would be appreciated as part of my job is to support open-science and development of softwares for the neuroscience community. I am working for the [Human Neuroscience Platform](https://hnp.fcbg.ch/) (HNP) from the Fondation Campus Biotech Geneva (FCBG). Let me check today or tomorrow for a logo in a suitable format.

**Comment by mscheltienne** [comment]
@sappelhoff Do you mind if I push the logo/changes directly to this PR or would you prefer a link?

**Comment by sappelhoff** [comment]
> @sappelhoff Do you mind if I push the logo/changes directly to this PR or would you prefer a link?

please feel free to push! My main objective was to get the ball rolling for adding you to the list :-)

**Comment by larsoner** [comment]
Thanks @sappelhoff @mscheltienne !

File: doc/overview/people.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ Steering Council
 * `Guillaume Favelier`_
 * `Luke Bloy`_
 * `Mainak Jas`_
+* `Mathieu Scheltienne`_
 * `Marijn van Vliet`_
```
Comment (drammock) [inline doc/overview/people.rst:31]: just a note that alphabetical order was violated here. Open invitation for someone to fix it when a hefty script is running or when they're stuck on some other problem.

Comment (sappelhoff) [reply doc/overview/people.rst:31]: fixed in b0f8ce69c7258319624aebdd6de63b23e1c78ad5

**Pull request merged**